src/aosm/azext_aosm/configuration_models/onboarding_vnf_input_config.py

```diff
@@ -233,7 +233,7 @@ def validate(self):
             raise ValidationError("You must include at least one image")
         for image in self.images:
             (_, _, version) = split_image_path(image)
-            if not is_semver(version):
+            if not is_valid_nexus_image_version(version):
                 raise ValidationError(f"{image} has invalid version '{version}'.\n"
                                       "Allowed formats are major.minor.patch")

```

Comment:
Suggest changing 
"Allowed formats are major.minor.patch" to
"Allowed format is major.minor.patch"